### ISSUE TITLE
Tray icon leaks wallet name and balance

### ISSUE THREAD

**Issue by euser991** [spesmilo/electrum#5665]
In Windows, even after programs (including Electrum) are closed, you can still see the tray icon information for all programs you've used before. In Windows 7 you find this by expanding the tray and clicking the "Customize..." link. In Windows 10 you can open the start menu and search for "Select which icons appear on the taskbar".
Since the Electrum tray icon contains the wallet name and BTC balance, this information can be found even if Electrum is closed and your wallet is encrypted. Even if you use the portable version on an encrypted drive that's no longer connected to the system.
The BTC balance (and preferably the wallet name too) should be removed from the tray icon.
In the portable version the tray icon should be removed altogether. Portable apps should not leak information and leave traces of themselves on the system.

![trayicon](https://user-images.githubusercontent.com/55965517/65844188-b4040180-e324-11e9-8ee9-ad94c2aa0cea.png)

**Comment by SomberNight** [comment]
>  In the portable version the tray icon should be removed altogether. Portable apps should not leak information and leave traces of themselves on the system.

On a closed source proprietary operating system, this seems like an impossibly high standard.
The most we could do, if we wanted to, is "best effort" towards this.
E.g. the Qt gui sends OS notifications about new transactions; do those leave a trace?
My first reaction would be to say having to deal with all this is out of scope, and using the portable version offers no guarantees about leaving "no traces".

> Since the Electrum tray icon contains the wallet name and BTC balance, this information can be found even if Electrum is closed and your wallet is encrypted.

Yes indeed, this is annoying, thanks for pointing it out.

**Comment by SomberNight** [comment]
Currently the tray icon text is the same as the "status text" at the bottom left of the main window:
https://github.com/spesmilo/electrum/blob/1773bd6cd6b9aaa693c6c1cb865f88fc3597e09f/electrum/gui/qt/main_window.py#L882
I guess it could be changed to "Electrum 3.3.8", for instance.

**Comment by wartjugger** [comment]
Hiding the balance makes some sense, less so hiding the wallet name.
The wallet name is the file name that holds the wallet info. This is visible to anyone using the system anyway. Now if you're working with multiple wallets in parallel, having the wallet name as part of the tooltip is actually very useful.
